squidb-addons/squidb-support-loader/src/com/yahoo/squidb/support/SquidCursorLoader.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2015, Yahoo Inc.
+ * Copyrights licensed under the Apache 2.0 License.
+ * See the accompanying LICENSE file for terms.
+ */
+package com.yahoo.squidb.support;
+
+import android.support.v4.content.AsyncTaskLoader;
+import android.content.Context;
+import android.support.v4.content.CursorLoader;
+import android.net.Uri;
+
+import com.yahoo.squidb.data.AbstractModel;
+import com.yahoo.squidb.data.SquidCursor;
+import com.yahoo.squidb.data.SquidDatabase;
+import com.yahoo.squidb.sql.Query;
+
+/**
+ * A {@link CursorLoader} that queries a {@link SquidDatabase}. This version of SquidCursorLoader uses the support
+ * library instead of the base Android SDK.
+ */
+public class SquidCursorLoader<T extends AbstractModel> extends AsyncTaskLoader<SquidCursor<T>> {
```

Comment:
I might be in favor of naming it SquidSupportCursorLoader. Granted the Android way is the support library classes have the same name but different package qualifier as the normal ones, but that lends itself to mistakes IMO.